main/main.go

```diff
@@ -278,14 +280,19 @@ func maybeSetEnv(key, value string) {
 func main() {
 	argParse()
 
-	difftool := NewDiffTool()
+	difftool, err := NewDiffTool()
 
 	if len(options.remoteClusterName) > 0 {
+		if err != nil {
+			fmt.Printf("Error creating difftool: %v\n", err)
+			os.Exit(1)
+		}
```

Comment:
this error handling needs to be outside of the remoteClusterName check

Reply:
done